google-cloud-bigtable/src/main/java/com/google/cloud/bigtable/data/v2/BigtableDataClient.java

```diff
@@ -247,6 +250,46 @@ public void readRowsAsync(Query query, ResponseObserver<Row> observer) {
     return stub.createReadRowsCallable(rowAdapter);
   }
 
+  /**
+   * Convenience method to asynchronously mutate a single row atomically. Cells already present in
+   * the row are left unchanged unless explicitly changed by the {@link RowMutation}.
+   *
+   * <p>Sample code:
+   *
+   * <pre>{@code
+   * InstanceName instanceName = InstanceName.of("[PROJECT]", "[INSTANCE]");
+   * try (BigtableClient bigtableClient = BigtableClient.create(instanceName)) {
+   *   RowMutation mutation = RowMutation.create("[TABLE]", "[ROW KEY]")
+   *     .setCell("[FAMILY NAME]", "[QUALIFIER]", "[VALUE]");
+   *
+   *   ApitFuture<Void> future = bigtableClient.mutateRow(mutation);
+   * }
+   * }</pre>
+   */
+  public ApiFuture<Void> mutateRow(RowMutation rowMutation) {
+    return mutateRowCallable().futureCall(rowMutation);
+  }
+
+  /**
+   * Mutates a single row atomically. Cells already present in the row are left unchanged unless
+   * explicitly changed by the {@link RowMutation}.
+   *
+   * <p>Sample code:
+   *
+   * <pre>{@code
+   * InstanceName instanceName = InstanceName.of("[PROJECT]", "[INSTANCE]");
+   * try (BigtableClient bigtableClient = BigtableClient.create(instanceName)) {
+   *   RowMutation mutation = RowMutation.create("[TABLE]", "[ROW KEY]")
+   *     .setCell("[FAMILY NAME]", "[QUALIFIER]", "[VALUE]");
+   *
+   *   bigtableClient.mutateRowCallable.call(mutation);
```

Comment:
syntax (need parens for the method call)

Reply:
fixed